front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -398,6 +399,21 @@ function Suggestions({
     };
   }, [instructions, owner, suggestions]);
 
+  const [showLeftGradients, setshowLeftGradients] = useState(false);
```

Comment:
Would isolate that logic into a separate component, HorizontallyScrollableDiv, for readability
Can take care of it you're fine with that